### PR TITLE
BCI labels

### PR DESCRIPTION
**Pull request by dmpop**
Fix BCI-157

### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/containers-get.xml [hunk 1]
```diff
@@ -303,13 +303,84 @@
...
+      </para>
+<screen>podman inspect registry.suse.com/bci/bci-base:15.4 | \
+jq '.[0].Labels'</screen>
+      <para>
+        It is also possible to retrieve the value of a specific label:
+      </para>
+<screen>podman inspect registry.suse.com/bci/bci-base:15.4 | \
+jq '.[0].Labels["com.suse.sle.base.supportlevel"]'</screen>
+      <para>
+        The preceding command retrieve the value of the <literal>com.suse.sle.base.supportlevel</literal> label.
```
Comment (dcermak) [inline xml/containers-get.xml:367]: or `retrieved`
```suggestion
        The preceding command retrieves the value of the <literal>com.suse.sle.base.supportlevel</literal> label.
```

File: xml/containers-get.xml [hunk 2]
```diff
@@ -303,13 +303,84 @@
...
+      </para>
+      <para>
+        The <package>scope</package> tool makes it possible to examine labels of an image without pulling it first. For example:
```
Comment (dcermak) [inline xml/containers-get.xml:370]:
```suggestion
        The <package>skopeo</package> tool makes it possible to examine labels of an image without pulling it first. For example:
```

File: xml/containers-get.xml [hunk 3]
```diff
@@ -303,13 +303,84 @@
...
+    </sect3>
+    </sect2>
 
   <sect2 xml:id="sec-tags">
    <title>Tags</title>
    <para>
     Tags are used to refer to images. A tag forms a part of the image's name.
-    Unlike labels, tags can be freely defined, and they are usually used to
+    Unlike labels, tags can be freely defined, and they are normally used to
     indicate a version number.
```
Comment (dcermak) [inline xml/containers-get.xml:384]: Labels can be freely defined as well.

**Review by dcermak — COMMENTED** [review]
LGTM, although I'd suggest to add possible values for the release-stage and supportlevel labels.
Also, the lifecycle url label is missing, which is to a certain extent also important

File: xml/containers-get.xml [hunk 4]
```diff
@@ -303,7 +303,108 @@
...
+      &bci;s feature the following labels.
+    </para>
+    <variablelist>
+    <varlistentry>
+    <term>
+      com.suse.image-type
+    </term>
+    <listitem>
+    <para>
+      Indicates that the container image belongs to the &sle; product family.
```
Comment (dcermak) [inline xml/containers-get.xml:318]: This one is not correct. It identifies whether this is a pure BCI or an application container based on another BCI

File: xml/containers-get.xml [hunk 5]
```diff
@@ -303,7 +303,108 @@
...
+      <para>
+        It is also possible to retrieve the value of a specific label:
+      </para>
+<screen>podman inspect registry.suse.com/bci/bci-base:15.4 | \
+jq '.[0].Labels["com.suse.sle.base.supportlevel"]'</screen>
+      <para>
+        The preceding command retrieves the value of the <literal>com.suse.sle.base.supportlevel</literal> label.
+      </para>
+      <para>
+        The <package>scopeo</package> tool makes it possible to examine labels of an image without pulling it first. For example:
```
Comment (dcermak) [inline xml/containers-get.xml:400]: Maybe add a link to skopeo as well?
```suggestion
        The <package>skopeo</package> tool makes it possible to examine labels of an image without pulling it first. For example:
```

**Review by dariavladykina — COMMENTED** [review]
LGTM!

**Review by dariavladykina — APPROVED** [review]
LGTM!

File: xml/containers-get.xml (hunk 5 above)
Comment (dcermak) [reply xml/containers-get.xml:400]: The package name is still incorrect. It is s**k**opeo not s**c**opeo

**Review by dcermak — APPROVED** [review]
LGTM, thanks!

**Comment by ZerwasMOD** [comment]
Looks good to me as well. Thanks Dima!

**Pull request merged**